lxd/instance_backup.go

```diff
@@ -615,8 +615,8 @@ func instanceBackupDelete(d *Daemon, r *http.Request) response.Response {
 		return nil
 	}
 
-	resources := map[string][]string{}
-	resources["container"] = []string{name}
+	resources := map[string][]api.URL{}
+	resources["container"] = []api.URL{*api.NewURL().Path(version.APIVersion, "container", name, "backups", backup.Render().Name)}
```

Comment:
Ah, good spot, this should be `instances` and then copied into `container`, like in `instanceBackupsPost`.